packages/client/tests/data/index.ts

```diff
@@ -30,7 +30,7 @@ export const db = factory({
 
     searchableMpeRooms: {
         roomID: primaryKey(() => datatype.uuid()),
-        roomName: () => random.words(datatype.number({ min: 1, max: 5 })),
+        roomName: () => random.words(),
```

Comment:
:thumbsup:

Reply:
should we be doing the same for the same file line 25 ?

Reply:
Why not! In another PR?